src/flake8_bugbear/plugins/redundant_tuple_in_exception_handler.rs

```diff
@@ -1,20 +1,35 @@
 use rustpython_ast::{Excepthandler, ExcepthandlerKind, ExprKind};
 
 use crate::ast::types::Range;
+use crate::autofix::Fix;
 use crate::check_ast::Checker;
 use crate::checks::{Check, CheckKind};
+use crate::code_gen::SourceGenerator;
 
 /// B013
 pub fn redundant_tuple_in_exception_handler(checker: &mut Checker, handlers: &[Excepthandler]) {
     for handler in handlers {
         let ExcepthandlerKind::ExceptHandler { type_, .. } = &handler.node;
         if let Some(type_) = type_ {
             if let ExprKind::Tuple { elts, .. } = &type_.node {
-                if elts.len() == 1 {
-                    checker.add_check(Check::new(
-                        CheckKind::RedundantTupleInExceptionHandler(elts[0].to_string()),
+                if let [elt] = &elts[..] {
+                    let mut check = Check::new(
+                        CheckKind::RedundantTupleInExceptionHandler(elt.to_string()),
                         Range::from_located(type_),
-                    ));
+                    );
+                    if checker.patch(check.kind.code()) {
+                        let mut generator = SourceGenerator::new();
+                        if let Ok(()) = generator.unparse_expr(elt, 0) {
+                            if let Ok(content) = generator.generate() {
+                                check.amend(Fix::replacement(
```

Comment:
```python
try:
    pass
except (ValueError,):
    pass
```

is fixed to:

```python
try:
    pass
except (ValueError):
    pass
```

Is it possible to remove unnecessary parens?

Reply:
Yes

Reply:
I think it will require some improvements to the source code generator.

Reply:
I can take a look at it...

Reply:
Oh, it looks like the issue here is actually in the RustPython locations, which exclude the parens.